ui/components/Icon.tsx

```diff
@@ -167,6 +171,51 @@ function getIcon(i: IconType) {
     case IconType.PauseIcon:
       return PauseIcon;
 
+    case IconType.NotificationsBell:
+      return NotificationsBellIcon;
+
+    case IconType.SourcesIcon:
+      return () => (
+        <svg
```

Comment:
I've done this before, no problems for me here. I tend to throw them in their own component just to clean things up:

```jsx
export default function CoolIcon() {
  <svg >...</svg>
} 
```

If you add a `className` prop + styled-components wrapper they can be styled like everything else.

Reply:
Great will move to outside components